src/functions/template-tags/general.php

```diff
@@ -1344,8 +1322,24 @@ function tribe_events_get_the_excerpt( $post = null, $allowed_html = null ) {
 		 */
 		$excerpt_more = apply_filters( 'excerpt_more', ' [&hellip;]' );
 
-		// Now we actually trim it
-		$excerpt = wp_trim_words( $excerpt, $excerpt_length, $excerpt_more );
+		// Trim the excerpt. The following is just wp_trim_words() without its use of wp_strip_all_tags().
+		if ( strpos( _x( 'words', 'Word count type. Do not translate!', 'the-events-calendar' ), 'characters' ) === 0 && preg_match( '/^utf\-?8$/i', get_option( 'blog_charset' ) ) ) {
```

Comment:
This approach is a little odd and although you've already linked to `wp_trim_words()`, where people can find the rationale, I feel like it makes sense to have the same explanatory note to translators here in our own method, since A) it means devs don't need to jump to that function to figure out why we're doing this and B) as we're using our own text domain, it may help translators to figure out what's going on here. We wouldn't want someone to accidentally translate `'words'`, for instance.